7-properties.md

```diff
@@ -110,88 +110,47 @@ To make this work, IPFS has to solve a few problems:
 - [Portable Encodings](#755-portable-encodings) - using portable serialization formats
 - [Secure Communications](#756-secure-communication) - using ciphersuites to establish security and privacy (like TLS)
 
-### 7.5.1 Protocol-Multiplexing
+### 7.5.1 Stream Multiplexing
 
-Protocol Multiplexing means running multiple different protocols over the same stream. This could happen sequentially (one after the other), or concurrently (at the same time, with their messages interleaved). We achieve protocol multiplexing using three pieces:
+Stream Multiplexing is the process of multiplexing (or combining) many different streams into a single one. This is a complicated subject because it enables protocols to run concurrently over the same wire, and all sorts of notions regarding fairness, flow control, head-of-line blocking, etc. start affecting the protocols. In practice, stream multiplexing is well understood and there are many stream multiplexing protocols.
 
-- [multistream](#752-multistream-self-describing-protocol-stream) - self-describing protocol streams
-- [multistream-select](#753-multistream-selector-self-describing-protocol-stream-selector) - a self-describing protocol selector
-- [Stream Multiplexing](#754-stream-multiplexing) - running many independent streams over the same wire
+The multiplexing protocols used at the moment are:
 
-### 7.5.2 multistream - self-describing protocol stream
+- Mplex
+- [Yamux](https://github.com/hashicorp/yamux/blob/master/spec.md)
```

Comment:
- QUIC, which has stream multiplexing built into the protocol

Reply:
It is in the "Other ideas" below, since we're not actually using QUIC at the moment as far as I know.

Reply:
Turns out we're using it. Fixed.